### ISSUE TITLE
locale change is not working due to moment not being exported

### ISSUE THREAD

**Issue by panpiotrs** [alienfast/picker.js#13]
Hello, as in title: How can I change language for days and months?
http://codepen.io/anon/pen/JXeqNe
It's not changing even when I add specific moment.js locale:
http://codepen.io/anon/pen/ONaYxN
or whole moment.js with locales:
http://codepen.io/anon/pen/aNQrLR

**Comment by rosskevin** [comment]
I'll be back in a few hours and will look, but I'm expecting you include the momentjs locale file for your lang, and pass `lang: xx` in the options.

**Comment by rosskevin** [comment]
BTW thanks for the test case

**Comment by panpiotrs** [comment]
Yes, exactly that. This is the second link and now I can see that there is error in Chrome console when only locale file is added without whole moment.js library: "pl.js:36 Uncaught TypeError: Cannot read property 'defineLocale' of undefined"
But even with moment.js and without this bug (third link) it is still in default language.

**Comment by rosskevin** [comment]
So, it fails because the locale isn't part of the rollup-based iife file, so moment isn't currently globally defined.  I'll see if I can figure that out with the packaging.  I do expect you need to include the locale after the `picker.iife.js`, but that currently yields the same error.

**Comment by rosskevin** [comment]
@panpiotrs are you compiling this with others inside an asset pipeline or using it directly? 

This is still an issue regardless, but I wanted to mention the ES2015 way.
i.e. if you were using this with `gulp-pipeline` to do your own rollup, you could be doing:

```javascript
import 'moment'
import 'moment/locale/pl'
import 'picker.js'
```

In this case, I would expect it to all go well.  

This is still a problem for the iife/self executing use - I'll keep looking at it.

**Comment by rosskevin** [comment]
Ok, I've exported moment for configuration, the trick is to set `var moment = window.picker.moment` before adding the script for your locale.

See this sample:
http://alienfast.github.io/picker.js/bmd4-locale.html

**Comment by rosskevin** [comment]
I'm marking this complete since I've provided an example that is working. If I find any better ways to do this I'll update this issue.

**Comment by panpiotrs** [comment]
Great, it's working this way, thank you :)

**Comment by rosskevin** [comment]
@panpiotrs I just published a change in v0.1.11.  You no longer need the the extra `script` tag to setup the global variable.  The demo is updated as well.

**Comment by panpiotrs** [comment]
Cool, thanks :)
